packages/IntegrationTest/jest.config.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT License.
+ *
+ * @format
+ * @ts-check
+ */
+
+// For a detailed explanation regarding each configuration property, visit:
+// https://jestjs.io/docs/en/configuration.html
+
+module.exports = {
+  // The root directory that Jest should scan for tests and modules within
+  rootDir: process.cwd(),
+
+  // A list of paths to directories that Jest should use to search for files in
+  roots: ['<rootDir>/runner/'],
+
+  // The test environment that will be used for testing
+  testEnvironment: 'node',
+
+  // The pattern or patterns Jest uses to detect test files
+  testRegex: '.*\\.test',
+
+  // Default timeout of a test in milliseconds
+  testTimeout: 300000,
```

Comment:
>300000 [](start = 15, length = 6)

Office Eng sets timeout to10 minutes, would we want to synchronize with Office Eng here?

Reply:
Could you help me understand the motivation to match Office tooling here?